ipu-plugin/ipuplugin/cmd/rootcmd.go

```diff
@@ -142,9 +148,14 @@ var (
 			}).Info("Configurations")
 
 			brCtlr, brType := getBridgeController(bridgeName, bridgeType, ovsCliDir, ovsDbPath)
-			p4Client := getP4Client(p4pkg, p4rtbin, portMuxVsi, defaultP4BridgeName, brType)
+			p4rtIpPort, err := convertNameToIpAndPort(p4rtName)
```

Comment:
Comment to clarify that in the cluster, then we use localhost:9559 which won't work and we will error out in the waitForP4Pod